sdk/storage/azure-storage-blobs/test/blob_service_client_test.cpp

```diff
@@ -251,9 +251,17 @@ namespace Azure { namespace Storage { namespace Test {
     properties.Cors.emplace_back(corsRule);
 
     properties.DeleteRetentionPolicy.Enabled = true;
-    properties.DeleteRetentionPolicy.Days = 5;
+    properties.DeleteRetentionPolicy.Days = 7;
 
+    try
+    {
+      m_blobServiceClient.SetProperties(properties);
+    }
+    catch (StorageException&)
+    {
+    }
```

Comment:
Why are we doing this? Catching an exception here and then asserting the next call to `SetProperties` doesn't throw is a strange test behavior.